mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/entity/EntityHelper.java

```diff
@@ -214,4 +220,37 @@
    */
   @NonNull
   Delete deleteByPrimaryKey();
+
+  /**
+   * The keyspace used in the queries generated by this helper.
+   *
+   * <p>This is determined by the following rules:
+   *
+   * <ul>
+   *   <li>If the DAO that this helper belongs to was created with a keyspace (see {@link
+   *       DaoKeyspace}), use that;
+   *   <li>Otherwise, if {@link Entity#defaultKeyspace()} is set for the entity class, use that;
+   *   <li>Otherwise, return {@code null}.
+   * </ul>
+   */
+  @Nullable
+  CqlIdentifier getKeyspaceId();
+
+  /**
+   * The table used in the queries generated by this helper.
+   *
+   * <p>This is determined by the following rules:
+   *
+   * <ul>
+   *   <li>If the DAO that this helper belongs to was created with a table (see {@link DaoTable}),
+   *       use that;
+   *   <li>Otherwise, if the entity class is annotated with {@link CqlName}, use that;
+   *   <li>Otherwise, use the name of the entity class, transformed by its {@link NamingStrategy}.
```

Comment:
javadoc plugin doesn't like that ul isn't closed:

```
10:55:42 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:jar (attach-javadocs) on project java-driver-mapper-runtime: MavenReportException: Error while generating Javadoc:
10:55:42 [ERROR] Exit code: 1 - /home/jenkins/workspace/datastax.java-driver.java2289.default/736569c2/mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/entity/EntityHelper.java:244: error: element not closed: ul
10:55:42 [ERROR] * <ul>
```

Reply:
That's an oversight, will fix.